kolibri/plugins/facility/assets/src/views/CsvInfoModal.vue

```diff
@@ -169,10 +169,15 @@
       yearInfo: 'A four-digit year, greater than 1900',
       listClassesEnrolled: 'Classes to enroll the user in as a learner:',
       listClassesEnrolledL1: 'Can be any type of user',
-      listClassesEnrolledL2: 'List of class names, separated by commas',
-      listClassesEnrolledL3: 'If an existing class does not match by name, it will be created',
-      listClassesAssigned: 'Classes to assign the user to as a coach:',
-      listClassesAssignedL1: 'Valid only for Coach and Admin user types',
+      listClassesEnrolledL2: 'Write the class names separated by commas',
+      listClassesEnrolledL3:
+        'If an existing class does not match by name, a new class with that name  it will be created',
```

Comment:
This might have been an error on our side, and it made copy-pasting difficult. Replace with below:
``If an existing class does not match by name, a new class with that name will be created``